src/main/java/teammates/ui/controller/ActionFactory.java

```diff
@@ -109,6 +109,8 @@
         map(INSTRUCTOR_STUDENT_RECORDS_PAGE, InstructorStudentRecordsPageAction.class);
         map(INSTRUCTOR_STUDENT_RECORDS_AJAX_PAGE, InstructorStudentRecordsAjaxPageAction.class);
 
+        map(INSTRUCTOR_STUDENT_LIST_REMIND, InstructorStudentListRemindAction.class);
```

Comment:
This should be put above, following the alphabetical order

Reply:
Thank you for noting this. I am fixing this as you suggested.